src/styles/main-theme.ts

```diff
@@ -0,0 +1,17 @@
+export interface Theme {
+  fontFamily: string;
+}
```

Comment:
We can leverage type extension as [described in s-c docs](https://styled-components.com/docs/api#create-a-declarations-file).

If you create `declarations/styled-components.d.ts` with the following contents:

```ts
import "styled-components";

declare module "styled-components" {
  export interface DefaultTheme {
    fontFamily: string;
  }
}
```

it will be possible to access theme in a type-safe way:

<img width="701" alt="Screenshot 2022-01-10 at 21 15 31" src="https://user-images.githubusercontent.com/608862/148840838-7b41f3d2-7045-4b1e-9260-17cdf7a1ad85.png">

This will be also possible:

```ts
import { DefaultTheme } from "styled-components";

const mainTheme: DefaultTheme = {
   font-family: "..."'
}
```